quic/s2n-quic-transport/src/recovery/manager.rs

```diff
@@ -37,7 +37,7 @@ pub struct Manager<Config: endpoint::Config> {
     //= https://www.rfc-editor.org/rfc/rfc9002#section-A.3
     //# An association of packet numbers in a packet number space to information about them.
     //  These are packets that are pending acknowledgement.
-    sent_packets: SentPackets,
+    sent_packets: SentPackets<<<Config::CongestionControllerEndpoint as congestion_controller::Endpoint>::CongestionController as congestion_controller::CongestionController>::PacketInfo>,
```

Comment:
is it possible to use the macro here also?

Reply:
No, because it would prevent using `derive`, as that doesn't work when there is a macro in a type. You get this error:

> `derive` cannot be used on items with type macros